### ISSUE TITLE
[Keyboard Navigation – Make Code - Publish project] Multiple controls are not accessible via keyboard present under 'Share project' pop-up window.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3605]
**User Experience**:
User rely on keyboard will find difficulty to interact with the control if that control is not accessible through keyboard as user won't be able to perform functionality over such control.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Share" option appearing in header section and select it.
6. Navigate to "Publish project" under  "Share Project" dialog and select it.
7. Navigate to all the control appearing like QR Code, Embed(Code, Editor and Simulator).
8. Verify that controls like(Facebook, Twitter, Post to Forum & Bar code) are accessible through keyboard or not.

**Actual Result:**
Controls like(Facebook, Twitter, Post to Forum & Bar code) are not accessible through keyboard under 'Share project' pop-up and due to this, user is unable to interact with such control and won’t be able to perform functionality over such controls.

**Expected Result:**
Controls like(Facebook, Twitter, Post to Forum & Bar code) should be accessible through keyboard under 'Share project' pop-up, so that user could easily interact with these controls and could perform functionality over these controls.

**Note:**
Same issue appear under Chrome & Firefox.

**WCAG Reference:**
[WCAG 2.1.1 - Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev1, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG2.1.1, #Win10-Edge(Chromium)

[WCAG2.1.1_Publish project.zip](https://github.com/microsoft/pxt-microbit/files/5499452/WCAG2.1.1_Publish.project.zip)

**Comment by jwunderl** [comment]
social buttons need sui.fireClickOnEnter
